data/gas_services.json

```
@@ -69,6 +69,10 @@
     {
       "id": "arbitrum",
       "address": "0xbE406F0189A0B4cf3A05C286473D23791Dd44Cc6"
+    },
+    {
+      "id": "kava",
+      "address": "deployment-pending"
```

```suggestion
      "address": "0xbE406F0189A0B4cf3A05C286473D23791Dd44Cc6"
```
deploying it soon, so keeping it ready